src/utils/userAuthentication.js

```diff
@@ -46,7 +46,7 @@ const hashPassword = async (plaintextPassword) => {
 };
 
 const checkAuthenticated = (req, res, next) => {
-  if (req.isAuthenticated()) {
+  if (req.isAuthenticated() || config.NODE_ENV !== 'production') {
```

Comment:
I thought this would be helpful for testing stuff. Let me know what you guys think

Reply:
Yeah I'm a big fan of this